<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
color: rgb(234, 184, 255);</string>
  </property>
  <widget class="QPushButton" name="m_btn_seoul">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(45, 63, 185);
border:0px;</string>
   </property>
   <property name="text">
    <string>서울</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_busan">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
selection-background-color: rgb(44, 56, 97);
color: rgb(255, 255, 255);
background-color: rgb(45, 63, 185);
border:0px;</string>
   </property>
   <property name="text">
    <string>부산</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_dejun">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>370</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(45, 63, 185);
border:0px;</string>
   </property>
   <property name="text">
    <string>대전</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_jeju">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(45, 63, 185);
border:0px;</string>
   </property>
   <property name="text">
    <string>제주</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ham">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>470</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(45, 63, 185);
border:0px;</string>
   </property>
   <property name="text">
    <string>함양</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
